src/main/resources/assets/spectritemod/lang/de_de.lang

```diff
@@ -0,0 +1,228 @@
+iteminfo.spectrite_sword_special.l2=§7Sofortschaden II
+iteminfo.spectrite_sword_special.l3=§7Giftschaden II@
+iteminfo.spectrite_sword_2.l1=§7Alle #s:
+iteminfo.spectrite_sword_2.l2=§7Sofortschaden III
+iteminfo.spectrite_sword_2.l3=§7Giftschaden III@
+iteminfo.spectrite_sword_2_special.l1=§7Alle #s:
+iteminfo.spectrite_sword_2_special.l2=§7Sofortschaden IV
+iteminfo.spectrite_sword_2_special.l3=§7Giftschaden IV@
+iteminfo.spectrite_helmet.0.l1=§7Maximale Gesundheit +1
+iteminfo.spectrite_helmet.0.l2=§7Bonus festlegen:
+iteminfo.spectrite_helmet.0.l3=§7Regeneration I@
+iteminfo.spectrite_chestplate.0.l1=§7Maximale Gesundheit +2
+iteminfo.spectrite_chestplate.0.l2=§7Bonus festlegen:
+iteminfo.spectrite_chestplate.0.l3=§7Regeneration I@
+iteminfo.spectrite_leggings.0.l1=§7Maximale Gesundheit +1.5
+iteminfo.spectrite_leggings.0.l2=§7Bonus festlegen:
+iteminfo.spectrite_leggings.0.l3=§7Regeneration I@
+iteminfo.spectrite_boots.0.l1=§7Maximale Gesundheit +0.5
+iteminfo.spectrite_boots.0.l2=§7Bonus festlegen:
+iteminfo.spectrite_boots.0.l3=§7Regeneration I@
+iteminfo.spectrite_helmet.1.l1=§7Maximale Gesundheit +2
+iteminfo.spectrite_helmet.1.l2=§7Bonus festlegen:
+iteminfo.spectrite_helmet.1.l3=§7Regeneration I@
+iteminfo.spectrite_chestplate.1.l1=§7Maximale Gesundheit +4
+iteminfo.spectrite_chestplate.1.l2=§7Bonus festlegen:
+iteminfo.spectrite_chestplate.1.l3=§7Regeneration I@
+iteminfo.spectrite_leggings.1.l1=§7Maximale Gesundheit +3
+iteminfo.spectrite_leggings.1.l2=§7Bonus festlegen:
+iteminfo.spectrite_leggings.1.l3=§7Regeneration I@
+iteminfo.spectrite_boots.1.l1=§7Maximale Gesundheit +1
+iteminfo.spectrite_boots.1.l2=§7Bonus festlegen:
+iteminfo.spectrite_boots.1.l3=§7Regeneration I@
+iteminfo.spectrite_helmet.2.l1=§7Maximale Gesundheit +4
+iteminfo.spectrite_helmet.2.l2=§7Bonus festlegen:
+iteminfo.spectrite_helmet.2.l3=§7Regeneration II@
+iteminfo.spectrite_chestplate.2.l1=§7Maximale Gesundheit +8
+iteminfo.spectrite_chestplate.2.l2=§7Bonus festlegen:
+iteminfo.spectrite_chestplate.2.l3=§7Regeneration II@
+iteminfo.spectrite_leggings.2.l1=§7Maximale Gesundheit +6
+iteminfo.spectrite_leggings.2.l2=§7Bonus festlegen:
+iteminfo.spectrite_leggings.2.l3=§7Regeneration II@
+iteminfo.spectrite_boots.2.l1=§7Maximale Gesundheit +2
+iteminfo.spectrite_boots.2.l2=§7Bonus festlegen:
+iteminfo.spectrite_boots.2.l3=§7Regeneration II@
+iteminfo.spectrite_helmet.3.l1=§7Maximale Gesundheit +6
+iteminfo.spectrite_helmet.3.l2=§7Bonus festlegen:
+iteminfo.spectrite_helmet.3.l3=§7Schnelligkeit I
+iteminfo.spectrite_helmet.3.l4=§7Stärke I
+iteminfo.spectrite_helmet.3.l5=§7Resistenz I
+iteminfo.spectrite_helmet.3.l6=§7Regeneration III
+iteminfo.spectrite_helmet.3.l7=§7Feuerresistenz I@
+iteminfo.spectrite_chestplate.3.l1=§7Maximale Gesundheit +12
+iteminfo.spectrite_chestplate.3.l2=§7Bonus festlegen:
+iteminfo.spectrite_chestplate.3.l3=§7Schnelligkeit I
+iteminfo.spectrite_chestplate.3.l4=§7Stärke I
+iteminfo.spectrite_chestplate.3.l5=§7Resistenz I
```

Comment:
Not sure what "resistence" does in game. If it is similar to minecraft's "protection" enchantment then it may need to be renamed to something like "Schutz".